src/filter.js

```diff
@@ -223,19 +213,6 @@ define(function (require) {
    *  @method connect
    *  @param  {Object} unit
    */
-  p5.Filter.prototype.connect = function(unit) {
-    var u = unit || p5.soundOut.input;
-    this.output.connect(u);
-  };
-
-  /**
-   *  Disconnect all output.
-   *  
-   *  @method disconnect
-   */
-  p5.Filter.prototype.disconnect = function() {
-    this.output.disconnect();
-  };
 
   p5.Filter.prototype.dispose = function() {
     // remove reference from soundArray
```

Comment:
cool! i wasn't sure if I could add to the Effect.prototype.dispose in objects that extended it.